detectors/node/opentelemetry-resource-detector-alibaba-cloud/src/index.ts

```diff
@@ -14,4 +14,4 @@
  * limitations under the License.
  */
 
-export * from './detectors';
+export { alibabaCloudEcsDetector } from './detectors';
```

Comment:
note to reviewer: little step forward to resolve https://github.com/open-telemetry/opentelemetry-js/issues/4186